automated install/basic-install.sh

```diff
@@ -138,8 +145,68 @@ show_ascii_berry() {
 }
 
 # Compatibility
+
+53probe (){
+# Probe localhost via 127.0.0.1 for open port 53 and store process name if port 53 is occupied
+  if (echo > /dev/tcp/127.0.0.1/53) >/dev/null 2>&1; then
+    # what process is using 53?
+    who53="$(${SUDO} lsof -i :53 +c 0 | awk 'FNR==2{ print $1 }')"
+  fi
+}
+
+53check(){
+  # Print info regarding probing for open port 53
+  echo -e "${OVER}  ${INFO} Testing for port 53 availability...${COL_NC}"
+  53probe
+  # check running process and see if it's blank
+  if [ -z "$who53" ]; then
+    echo -e "  ${TICK} ${COL_LIGHT_GREEN}Port 53 is available.${COL_NC}"
+    # check running process and see if it's pihole-FTL
+  elif [ "$who53" = "pihole-FTL" ]; then
+    # proceed with install
+    echo -e "${OVER}  ${TICK} Port 53 is in use by our resolver ${COL_LIGHT_GREEN}($who53)${COL_NC}, proceeding with setup"
+  elif [ "$who53" = "dnsmasq" ]; then
+    # If dnsmasq is present, set the dnsmasq-flag to true for future reference
+    # (after packages and dependencies are installed).
+    # dnsmasq will be disabled at end of install, prior to FTLDNS start.
+    echo -e "${OVER}  ${EXCL} Port 53 is in use by ${COL_LIGHT_RED}$who53${COL_NC}. 
+    The installer will disable ${COL_LIGHT_RED}$who53${COL_NC} after dependencies and packages
+    have been downloaded, and replace the system DNS resolver with FTLDNS."
+    dnsmasq_flag=true
+  elif [ "$who53" = "systemd-resolve" ]; then
+    # If systemd-resolved is present, set the systemd-resolved-flag to true for future reference
+    # (after packages and dependencies are installed).
+    # systemd-resolved will be disabled at end of install, prior to FTLDNS start.
+    echo -e "${OVER}  ${EXCL} Port 53 is in use by ${COL_LIGHT_RED}$who53${COL_NC}.
+      The installer will disable ${COL_LIGHT_RED}$who53${COL_NC} after dependencies and packages
+      have been downloaded, and replace the system DNS resolver with FTLDNS."
+      systemd_resolved_flag=true
+  else
+    # port 53 is used by something else, stop install
+    echo -e "${OVER}  ${EXCL} ${COL_LIGHT_YELLOW}WARNING: Port 53 (mandatory for FTLDNS) is already in use by ${COL_LIGHT_RED}$who53${COL_NC}.
+      Since this will interfere with the functionality of FTLDNS, the installer cannot continue.
+      Please visit ${COL_LIGHT_CYAN}https://discourse.pi-hole.net/t/ftldns-pi-holes-own-dns-dhcp-server/${COL_NC}
```

Comment:
Why should people visit this specific thread for this issue? It does not help them remove whatever is using the port.

Reply:
It was relevant for the context that was quoted in (as in a known issue that is addressing the poster's issue).

Reply:
That thread is just an announcement for FTLDNS, not a place where they can get help. Redirecting them to the Discourse homepage would be a bit better.